xapian-core/include/xapian/query.h

```diff
@@ -211,6 +211,14 @@ class XAPIAN_VISIBILITY_DEFAULT Query {
 	 */
 	OP_VALUE_GE = 11,
 
+    /** Match only documents where a value slot is > a given value.
+	 *
+	 *  Similar to @a OP_VALUE_RANGE, but open-ended.
+	 *
+	 *  This operator never contributes weight.
+	 */
+	OP_VALUE_GT = 17,
```

Comment:
It would be better to keep them in numerical order, I think. That could then be the same order you put them in switch or chained else/if statements.